### PR TITLE
Allow custom ops to return an array of tensors

### PR DESCRIPTION
**Pull request by matthewfl**
This PR adds support to `fx_importer.py` for handling custom ops that return an array of tensors.  As long as the length of the array is consistent across runs (determined statically), then this patch will work.  This does not require that the number of tensors returned is determined by the op's definition.

CC @sjain-stanford

### CODE REVIEW

File: test/python/fx_importer/custom_op_test.py
Comment (sjain-stanford) [inline test/python/fx_importer/custom_op_test.py:119]: Could you also add `dynamic_shapes` on this for testing shape expressions and custom ops with multiple outputs?

**Review by sjain-stanford — COMMENTED** [review]
Looping in @stellaraccident for review as this touches fx_importer and might need further validation in the downstream suites Stella's team uses.

File: test/python/fx_importer/custom_op_test.py
Comment (matthewfl) [reply test/python/fx_importer/custom_op_test.py:130]: done

**Review by stellaraccident — APPROVED** [review]
Trickier than I would have expected but lgtm.

**Comment by sjain-stanford** [comment]
Thanks Stella. Going to land this as @matthewfl may not have write access.

**Pull request merged**
